lib/common/client/middleware/response/facility_parser.rb

```diff
@@ -29,7 +31,7 @@ def build_facility_attributes(location, mapping)
             facility.merge!(make_address_mappings(location, mapping))
             facility.merge!(make_benefits_mappings(location, mapping, facility['unique_id'])) if mapping['benefits']
             facility.merge!(make_hours_mappings(location, mapping))
-            facility.merge!(make_service_mappings(location, mapping)) if mapping['services']
+            facility.merge!(make_service_mappings(location, mapping, facility['unique_id'])) if mapping['services']
```

Comment:
I realize this isn't something you introduced, but I wonder if this would be clearer if `make_service_mappings` and/or the `services` map was was named `health_services`. It's probably just me but I keep forgetting the benefits services are handled separately. Maybe renaming to `make_benefits_mappings` and/or the `benefits` map to `benefits_services` would make sense too.

Reply:
+1 to this

Reply:
This is a good suggestion. This whole thing is a bit tangled and tough to follow. Might be good to take a longer look at how these fields are merged together at some point.